File: order.cpp
#include <iostream>
#include <curl/curl.h>

void sendOrder(std::string name, std::string product) {
    CURL *curl;
    CURLcode res;

    std::string webhook_url = "https://discord.com/api/webhooks/1349850414543142912/your-webhook";
    std::string message = "{\"username\":\"Goodboy Shop\",\"content\":\"🛒 **Neue Bestellung erhalten!**\\n**Produkt:** " + product + "\\n**Besteller:** " + name + "\"}";

    curl = curl_easy_init();
    if(curl) {
        curl_easy_setopt(curl, CURLOPT_URL, webhook_url.c_str());
        curl_easy_setopt(curl, CURLOPT_POST, 1);
        curl_easy_setopt(curl, CURLOPT_POSTFIELDS, message.c_str());
        res = curl_easy_perform(curl);
        curl_easy_cleanup(curl);
    }
}

int main() {
    sendOrder("Max Mustermann", "GTA Online Money Hack");
    return 0;
}

File: order.py
import sqlite3
import requests
import datetime

# Webhook URL
WEBHOOK_URL = "https://discord.com/api/webhooks/1349850414543142912/your-webhook"

# Verbindung zur Datenbank
conn = sqlite3.connect('shop.db')
cursor = conn.cursor()

# Tabelle für Bestellungen erstellen
cursor.execute('''CREATE TABLE IF NOT EXISTS orders (
                    id INTEGER PRIMARY KEY AUTOINCREMENT,
                    name TEXT,
                    product TEXT,
                    date TEXT)''')

# Bestellung speichern & senden
def order_product(name, product):
    date = datetime.datetime.now().strftime("%d.%m.%Y %H:%M")
    
    # Datenbank speichern
    cursor.execute("INSERT INTO orders (name, product, date) VALUES (?, ?, ?)", (name, product, date))
    conn.commit()

    # Webhook senden
    message = {
        "username": "Goodboy Shop",
        "embeds": [{
            "title": "🛒 **Neue Bestellung erhalten!**",
            "color": 16753920,
            "fields": [
                {"name": "🛍️ **Produkt:**", "value": f"**{product}**", "inline": False},
                {"name": "👤 **Besteller:**", "value": f"**{name}**", "inline": False},
                {"name": "📅 **Bestellzeit:**", "value": f"**{date}**", "inline": False}
            ]
        }]
    }
    
    requests.post(WEBHOOK_URL, json=message)
    print("✅ Bestellung erfolgreich gesendet!")

# Testfunktion
order_product("Max Mustermann", "FiveM Premium Mod Menü")
